firebase-firestore/src/main/java/com/google/firebase/firestore/remote/GrpcCallProvider.java

```diff
@@ -125,7 +125,7 @@ private ManagedChannel initChannel(Context context, DatabaseInfo databaseInfo) {
     // Wrap the ManagedChannelBuilder in an AndroidChannelBuilder. This allows the channel to
     // respond more gracefully to network change events (such as switching from cell to wifi).
     AndroidChannelBuilder androidChannelBuilder =
-        AndroidChannelBuilder.fromBuilder(channelBuilder).context(context);
```

Comment:
Intentional?

Reply:
Yes, `fromBuilder` is deprecated

https://github.com/grpc/grpc-java/blob/master/android/src/main/java/io/grpc/android/AndroidChannelBuilder.java#L90